aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -204,17 +204,30 @@ class QConvPackWeightInt8 final : public c10::OperatorKernel {
 
     auto weight_contig = weight.contiguous();
     auto bias_contig = bias.contiguous();
+    auto weight_zp = weight.q_zero_point() + 128;
+
+    int8_t* w_data = (int8_t*)weight_contig.data_ptr<c10::qint8>();
+    Tensor qnnp_weight = at::_empty_affine_quantized(
+        weight_contig.sizes(),
+        at::device(kCPU).dtype(kQUInt8),
+        weight.q_scale(),
+        weight_zp);
+    auto* qnnp_w_data = qnnp_weight.data_ptr<c10::quint8>();
+    for (int i = 0; i < weight_contig.numel(); ++i) {
+      qnnp_w_data[i] = static_cast<c10::quint8>(w_data[i] + 128);
+    }
     auto wt_ptr =
         guts::make_unique<PackedConvWeightsQnnp>(PackedConvWeightsQnnp{
             guts::make_unique<qnnpack::PrePackConvWeights>(
                 conv_p,
-                (uint8_t*)weight_contig.data_ptr<c10::quint8>(),
+                (uint8_t*)qnnp_w_data,
                 (int32_t*)bias_contig.data_ptr<c10::qint32>()),
-            weight_contig,
-            bias_contig,
+            weight_contig, /* int8_t weight */
+            bias_fp32.contiguous(), /* fp32 bias */
+            0, /* input_scale */
```

Comment:
add a comment that it's an intentionally "invalid" value - better yet, use c10::optional